specification/e-referrals-service-api.yaml

```diff
@@ -1681,6 +1681,103 @@ paths:
           $ref: 'components/schemas/responses/TooManyRequests.yaml'
         '500':
           $ref: 'components/schemas/responses/InternalServerError.yaml'   
+  /STU3/ReferralRequest/$ers.createFromCommunicationRequestActionLater:
+    post:
+      security:
+        - bearerAuth: []
+      description: |
+        ## Overview
+        Use this endpoint, as a service provider, to:
+          - convert the advice and guidance request you have received into a referral, AND
```

Comment:
I think we should be careful using advice and guidance request here as it's not used elsewhere in the xAPI, instead being referenced as a `CommunicationRequest`.